projects/core/mask/text-mask-options.ts

```diff
@@ -9,7 +9,7 @@ export interface TuiTextMaskOptions {
     /**
      * {@link https://github.com/text-mask/text-mask/blob/master/componentDocumentation.md#mask}
      */
-    mask: TuiTextMaskList | TuiTextMaskListHandler;
+    mask: TuiTextMaskList | TuiTextMaskListHandler | boolean;
```

Comment:
Hi!
thanks for PR.

Lets narrow `boolean` to just `false`.
`true` is not valid option, is it ?

Reply:
This is necessary in order to specify an empty mask value.
When using null, we get an error
![image (16)](https://user-images.githubusercontent.com/18559077/126612088-cc8789f2-c074-4675-84a6-1fcfb4ded863.png)
![image (15)](https://user-images.githubusercontent.com/18559077/126612101-61d74fac-a374-4f5b-ba26-7a8e0fc77fd0.png)
I found the solution here https://github.com/text-mask/text-mask/issues/435#issuecomment-285755605

Reply:
perhaps, we've misunderstood each other.

My suggestion was
```suggestion
    mask: TuiTextMaskList | TuiTextMaskListHandler | false;
```